Services/GEV/Reports/classes/class.gevEduBiographyGUI.php

```diff
@@ -224,19 +225,19 @@ protected function academyQuery(ilDate $start, ilDate $end) {
 	
 	protected function renderWBDPoints($tpl) {
 		require_once("Services/Calendar/classes/class.ilDatePresentation.php");
-		$user_utils = gevUserUtils::getInstance($this->target_user_id);
+		$uutils = gevUserUtils::getInstance($this->target_user_id);
 
 		$tpl->setVariable("WBD_SUM_TITLE", $this->lng->txt("gev_points_in_wbd"));
 		$tpl->setVariable("WBD_SUM_CERT_PERIOD_TITLE", $this->lng->txt("gev_points_in_wbd_cert_period"));
 		$tpl->setVariable("WBD_SUM_CUR_YEAR_TITLE", $this->lng->txt("gev_points_in_wbd_cert_year"));
 		$tpl->setVariable("WBD_SUM_CUR_YEAR_PRED_TITLE", $this->lng->txt("gev_points_at_end_of_cert_year"));
 		
-		$cy_start = $user_utils->getStartOfCurrentCertificationYear();
-		$cy_end = $user_utils->getStartOfCurrentCertificationYear();
+		$cy_start = $uutils->getStartOfCurrentCertificationYear();
```

Comment:
Warum sind die *CurrentCertificationYear-Methoden noch in UserUtils? Ich würden denken, die gehören auch zur WBD.